python/src/weightedexperiment_module.i

```diff
@@ -29,6 +29,7 @@
 %include BootstrapExperiment.i
 %include FixedExperiment.i
 %include GaussProductExperiment.i
+%include SmolyakExperiment.i
```

Comment:
as a remainder, this should go into the new experimental_module.i

Reply:
Fixed.

Reply:
I did that, but it seems to produce an error which prevents the code from building. It is related to the fact that the code uses a Collection of Experiments, that the compiler does not seem to find.
```
In file included from /home/devel/GIT/openturns-Smolyak/build/python/src/CMakeFiles/experimental.dir/experimental_modulePYTHON_wrap.cxx:4104:
/home/devel/GIT/openturns-Smolyak/python/src/openturns/PythonWrappingFunctions.hxx: In instantiation of ‘OT::Collection<T>* OT::buildCollectionFromPySequence(PyObject*, int) [with T = OT::WeightedExperiment; PyObject = _object]’:
/home/devel/GIT/openturns-Smolyak/build/python/src/CMakeFiles/experimental.dir/experimental_modulePYTHON_wrap.cxx:5495:88:   required from here
/home/devel/GIT/openturns-Smolyak/python/src/openturns/PythonWrappingFunctions.hxx:573:7: error: invalid use of incomplete type ‘struct OT::traitsPythonType<OT::WeightedExperiment>’
  573 |       check<typename traitsPythonType< T >::Type>(elt);
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/
```